sched/irq/irq_csection.c

```diff
@@ -192,208 +192,207 @@ irqstate_t enter_critical_section(void)
    * lists are valid.
    */
 
-  if (nxsched_get_initstate() >= OSINIT_TASKLISTS)
+  DEBUGASSERT(g_nx_initstate >= OSINIT_TASKLISTS);
```

Comment:
```suggestion

```

I think we can remove this assertion as it will be caught at line 332:

```
      cpu  = this_cpu();
      rtcb = current_task(cpu);
      DEBUGASSERT(rtcb != NULL);
```